test/powershell/Common/TestHostCS.psm1

```diff
@@ -191,7 +196,7 @@ namespace TestHost
 
         public override void WriteLine(string value)
         {
-            Streams.ConsoleOutput.Add(value);
+            Streams.ConsoleOutput.Add("::"+value+":NewLine");

```

Comment:
If color is specified, I would use `color:output` as the expected string to test the result, so why `::` is needed?